config/locales/member_facing.en.yml

```diff
@@ -150,10 +150,13 @@ en:
     you_will_be_calling: "We will call you and connect you to"
     instructions: "Just enter your phone number including the area code -- and we'll call shortly after."
     member_phone_number_title: 'Enter your phone number'
+    termination_message: "You haven't chosen a menu option, so the call will be terminated. Please try calling again if this wasn't intentional."
     text_to_speach_menu: "Please press one when you're ready and we will connect you. Press two to hear the campaigner's message again."
     errors:
       recaptcha_fail: "Sorry, but we're unable to make the call for you at this time."
       unknown: "Oops! Something went wrong, please try a different phone number or again in a few minutes."
+      akid: "Sorry, but for safety reasons, the calling tool is limited to recognized members who enter the website through a campaign email."
```

Comment:
I'd remove _recognized_ from the sentence so that it just says _"limited to members who enter [...] through a campaign email"_

Reply:
That's a good comment - I'm not actually sure what kind of copy would work well here now with the new behavior. It first recognizes the member by the cookie, and then falls back to the akid. So it's possible to take action not from a mailing, if the member is recognized. Maybe I should rephrase it to `members who have previously taken action with us`. But then someone could have taken action previously and might just not have the cookie. I think it might just be the easiest to say `limited to members whom we can recognize in the current session` or something like that. Maybe even just `recognized members`. What do you think would be best?